index.html

```diff
@@ -12,20 +12,26 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" type="text/css">
 </head>
 <body>
```

Comment:
- [ ] As previously requested, you still have the horizontal flow. Some sections are definitely wider than others or could be a margin on these sections in particular.

> I think it can be a great idea to fix the overflow-x on your page since it can create a bad UX (user experience)